python_ta/reporters/templates/template.html.jinja

```diff
@@ -111,8 +111,9 @@
                                 {% set occurrences = messages[0][message_id] %}
                             <div class="error-instance" id={{'{}-{}'.format(filenum, message_id)}}>
                                 <div class="info">
+                                    {% set mid = message_id.lower() %}
                                     <span class="error-link code-error-id">
-                                        <a href="http://www.cs.toronto.edu/~david/pyta/&#35;{{message_id}}" target="_blank">&#10064; {{ message_id }} ({{ occurrences[0].symbol }})  <span class="more-info">(Learn More)</span></a>
+                                        <a href="http://www.cs.toronto.edu/~david/pyta/checkers/index.html&#35;{{occurrences[0].symbol}}-{{mid}}" target="_blank">&#10064; {{ message_id }} ({{ occurrences[0].symbol }})  <span class="more-info">(Learn More)</span></a>
```

Comment:
Just use `mid` here, not the symbol name---this will make it consistent with the other links from the `doc` function. (Even though here we do have the name of the error, it's better to be consistent.)